### PR TITLE
actualize sub module, add headless_firefox

### PR DESCRIPTION
**Pull request by fravit4**

### CODE REVIEW

File: #1#input.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+! TRAVIS input file
```
Comment (romikoops) [inline #1#input.txt]: Do we need to have it?

File: spec/steps/categories/categories_maintaining_steps.rb [hunk 2]
```diff
@@ -47,6 +47,7 @@ module CategoryMaintainingSteps
 
   step 'I should not see category on categories list page' do
     category_name = category.name
+    sleep 2
```
Comment (romikoops) [inline spec/steps/categories/categories_maintaining_steps.rb]: We have to remove it. This is antipattern. If you create category via UI in action, then you have to check there with matcher, not in Verification and not with sleep

**Review by romikoops — CHANGES_REQUESTED** [review]
See my comments

File: #1#input.txt (hunk 1 above)
Comment (fravit4) [reply #1#input.txt]: deleted

File: spec/steps/categories/categories_maintaining_steps.rb (hunk 2 above)
Comment (fravit4) [reply spec/steps/categories/categories_maintaining_steps.rb]: replaced with 'is_expected.to have_no' matcher

**Review by romikoops — APPROVED** [review]

**Pull request merged**